src/algebra/indicator_function.lean

```diff
@@ -135,6 +135,28 @@ mul_indicator_eq_one.2 $ disjoint_empty _
 @[to_additive] lemma mul_indicator_empty' (f : α → M) : mul_indicator (∅ : set α) f = 1 :=
 mul_indicator_empty f
 
+lemma indicator_const_inverse_image {M} [has_zero M] (U : set α) (s : set M) (a : M) :
+  U.indicator (λ x, a) ⁻¹' s ∈ ({set.univ, U, Uᶜ, ∅} : set (set α)) :=
```

Comment:
Could you please prove the multiplicative version of this lemma? Also, is `set.mul_indicator_preimage` useful here?

Reply:
I added the multiplicative version and `to_additivized` it.  I also added a version of the lemma where there is only one case split.

I was not able to get `set.mul_indicator_preimage`, but moved the new lemmas just below it anyway.  I also updated the names via `inverse_image → preimage`.